digiwf-oce-cosmos-integration-service/src/main/java/io/muenchendigital/digiwf/ocecosmos/integration/api/configuration/OceCosmosConfiguration.java

```diff
@@ -17,6 +17,7 @@
 public class OceCosmosConfiguration {
 
     private static final String HEADER_PRINT_REQUEST = "requestPrint";
+    private static final String HEADER_PDF_PRINT_REQUEST = "requestPDFPrint";
```

Comment:
You could configure this in the application.properties.

e.g.
```
io.muenchendigital.digiwf.streaming.typeMappings.requestPDFPrint=requestPDFPrint
io.muenchendigital.digiwf.streaming.typeMappings.requestPrint=requestPrint
```

See Spring Cloud Stream Utils documentation https://github.com/it-at-m/digiwf-spring-cloudstream-utils#function-routing-configuration (Note: Spring Cloud Stream Utils moved to digiwf-core monorepo but the documenation is not migrated yet).

Reply:
I may add that in the future. Currently it doesn't have to be configurable